src/uct/base/uct_cm.c

```diff
@@ -151,8 +164,14 @@ ucs_status_t uct_cm_set_common_data(uct_cm_base_ep_t *ep,
 
     status = UCT_CM_SET_CB(params, UCT_EP_PARAM_FIELD_SOCKADDR_PACK_CB,
                            ep->priv_pack_cb, params->sockaddr_pack_cb,
-                           uct_cm_ep_priv_data_pack_callback_t,
-                           ucs_empty_function_return_invalid_param);
+                           uct_cm_ep_priv_data_pack_callback_t, NULL);
```

Comment:
because with new API user should define one from `sockaddr_pack_cb` or `cm_resolve_cb` (second one will be not set), IMHO checking against NULL-pointer for unset callback in the code is clearer than `ucs_empty_function_return_invalid_param`

Reply:
it was set to an empty function because of https://github.com/openucx/ucx/pull/5230#discussion_r433758866. but maybe now that it's either one of the 2 functions, it's ok to have it as null.